apis/nucleus/src/components/listbox/ListBox.jsx

```diff
@@ -214,6 +217,19 @@ export default function ListBox({
 
   const isVertical = listLayout !== 'horizontal';
   const count = layout.qListObject.qSize.qcy;
+
+  const PAGE_MAX_HEIGHT = 100;
+
+  const getCalculatedHeight = (ps) => {
```

Comment:
have you verified this on a use case that needs to page data and you scroll to where the data is filtered?
(My understanding is that `itemCount` should be the total number of items in the list not the number of items fetched the last time.)

Reply:
I have tried to get such a test case but for now we have to rely on intuition of how it works. What you say sounds reasonable. I just wonder what it would look like and if it's something we can live with for the first release – or if we could accumulate the height. Is `page.qArea.qTop` reliable for this?

Reply:
I would expect `page.qArea.qTop` and `page.qArea.qHeight` to be reliable.
but I rember hearing that engine did not have a reliable `page.qArea.qHeight` (in some versions? it is used in this PR)
so it may be best to check with the one who implemented the backend fix, on what the client can use.
(I have not had any direct communiction on what engine support in direct query apps)

Reply:
They also believe it should be reliable in DQ mode so let's proceed with this.